fix: update helm doc with NIM 2.18 and fix broken links (#260)
nginx/documentation: content/nim/deploy/kubernetes/deploy-using-helm.md

This guide explains how to deploy F5 NGINX Instance Manager on a Kubernetes or OpenShift cluster using Helm. You’ll learn how to download and use Docker images and customize your deployment.

> {{< note >}} Starting from NGINX Instance Manager 2.19, deploying NIM on the OpenShift cluster using Helm is supported. {{< /note >}}  

Reviewer: ```suggestion
{{< note >}} Starting in NGINX Instance Manager 2.19, you can deploy NGINX Instance Manager on an OpenShift cluster using Helm. {{< /note >}}  
```

Reviewer: FYI: We always need to write out product names. NIM --> NGINX Instnace Manager.

            tag: <version>
    ```

>     {{< note >}} Starting from NGINX Instance Manager 2.19, `secmon` pod is added to the NIM deployment. {{< /note >}}

Reviewer: ```suggestion
    {{< note >}} Starting in NGINX Instance Manager 2.19, the `secmon` pod is included in the NGINX Instance Manager deployment. {{< /note >}}
```
